blaserunning-and-shame/README.md

```diff
@@ -30,15 +30,27 @@ Shame is a uniquely exciting part of blaseball's rules: #watchparty on the offic
 
 ## Methodology and results
 
-Over a sample of 982 game logs, we looked for stolen bases, re-scored games, and determined whether those games ended in shame.
+Our sample consists of 982 game logs where the last out is present.
+Logs start in the middle of Season 2, and a number of games have incomplete logs due to various issues.
+
+On that sample, we looked for stolen bases, re-scored games, and determined whether those games ended in shame.
 The code for this research can be found in [shame.py](./shame.py), and source data can be found in `blaseball-archive-iliana.s3.us-west-2.amazonaws.com` (accessible via an S3 client; log timestamps 1596003134151 through 1596605456865).
 
 In this sample, 97 games (9.9%) resulted in shame under their original rule sets.
-Under new rules, 76 games (7.8%) resulted in shame; 29 games that originally resulted in shame no longer did, and 8 games now ended in shame that originally did not.
+Under new rules, 76 games (7.8%) resulted in shame; 25 games that originally resulted in shame no longer did, and 4 games now ended in shame that originally did not.
+The re-scored games are in Table 1.
+
+**Table 1 data in table1.csv**
 
-<!-- Possible TODO: Determine which teams were favored under this rule -->
+Out of the 25 affected games, 13 resulted in a new team winning, with 7 of those in the latter half of Season 2 where data is available.
```

Comment:
this sentence is kinda confusing.

Reply:
"Out of the 25 affected games, 13 results in a new team winning. Of those, seven occurred in the latter half of Season 2 (where data is available)." I think?